front/lib/api/assistant/actions/names.ts

```diff
@@ -3,7 +3,7 @@ export const DEFAULT_BROWSE_ACTION_NAME = "browse";
 export const DEFAULT_PROCESS_ACTION_NAME =
   "extract_structured_data_from_data_sources";
 export const DEFAULT_RETRIEVAL_ACTION_NAME = "search_data_sources";
-// The name below is used by the assistant builder when the user selects "Most recent data".
+// The name below is used by the assistant builder when the user selects "Include data".
 export const DEFAULT_RETRIEVAL_NO_QUERY_ACTION_NAME =
```

Comment:
Should we rename the action name presented to the model?

Reply:
Like cleared + we'll need the inclue name once we start using the action for JIT data source interactions cc @Fraggle